src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="Container">
+      <div className="Screen">0</div>
+      <div className="column">
+        <button type="button" className="white btn-style">AC</button>
+        <button type="button" className="white btn-style">+/-</button>
+        <button type="button" className="white btn-style">%</button>
+        <button type="button" className="orange btn-style">÷</button>
```

Comment:
- [ ] It would be nice if you have a child component in the **calculator.js** file because it is mentioned in the requirements that you should have at least one child component in your Calculator.js file :+1:

Hint: You can make this button component and import it here in the calculator.js file and use by giving props :blush: